module/data/activity/base-activity.mjs

```diff
@@ -613,20 +613,15 @@ export default class BaseActivityData extends foundry.abstract.DataModel {
 
   /**
    * Prepare the label for a compiled and simplified damage formula.
-   * @param {DamageData[]} parts  Damage parts to create labels for.
-   * @param {object} rollData     Deterministic roll data from the item.
+   * @param {object} rollData  Deterministic roll data from the item.
    */
-  prepareDamageLabel(parts, rollData) {
-    this.labels.damage = parts.map((part, index) => {
+  prepareDamageLabel(rollData) {
```

Comment:
Reasonably sure there are 3rd-party Activity implementations out there, so we probably need to provide a deprecation path for this signature change. Thankfully, I don't think it's possible that `rollData` would ever be an array, so checking if the first argument is an array should be a fairly straightforward check for a legacy invocation.